site/content/en/docs/tutorials/nvidia.md

```diff
@@ -19,6 +19,10 @@ date: 2018-01-02
 
 - Ensure you have an NVIDIA driver installed, you can check if one is installed by running `nvidia-smi`, if one is not installed follow the [NVIDIA Driver Installation Guide](https://docs.nvidia.com/datacenter/tesla/tesla-installation-notes/index.html)
 
+*Note: if you are using Windows WSL2, only install the driver on windows, and DO NOT install any linux nvidia driver. After instalation of windows driver, you many also need to execute `cp /usr/lib/wsl/lib/nvidia-smi /usr/bin/nvidia-smi` and `chmod ogu+x /usr/bin/nvidia-smi` in WSL2, because otherwise the nvidia-smi may not be found in PATH*
```

Comment:
can you make this a Tabbed View ? so the user Clicks on a tab and only reads their OS instructions ?